tb.go

```diff
@@ -0,0 +1,36 @@
+package merkletree
+
+import "github.com/coniks-sys/libmerkleprefixtree-go/crypto"
+
+type TemporaryBinding struct {
+	index []byte
+	value []byte
+	sig   []byte
+}
+
+func NewTemporaryBinding(pad *PAD, key string, value []byte) *TemporaryBinding {
+	str := pad.currentSTR
+	index := computePrivateIndex(key)
+	tb := str.SerializeWithSignature()
```

Comment:
This is a similar comment to what I had in https://github.com/coniks-sys/libmerkleprefixtree-go/pull/8#discussion_r70639337

In §3.3, `STR` is defined as,

> STR = SignSKd (t||tprev||roott ||H(STRprev)||P)

which to me look like it's just the signature.  I know I'm picking at nits, but now when I go look at the definition of `TB` in §4.1.1,

> TB = SignKd (STRt, i, k)

(which, in an aside to @masomel, probably wants to be, `TB = SignSKd (STRt || i || k)`, consistent with SKd and concatenation)

it's again not clear what is meant by `STR`.